minio/api.py

```diff
@@ -1558,7 +1559,7 @@ def _do_put_multipart_object(self, bucket_name, object_name, part_metadata,
 
         headers = {
             'Content-Length': part_metadata.size,
-            'Content-Md5': part_metadata.md5hasher.base64digest()
+            'Content-Md5': base64.b64encode(part_metadata.md5_hex.decode("hex"))
```

Comment:
For python3 , this does not work - I get an AttributeError: 'str' object has no attribute 'decode' part_metadata.md5_hex  is of type string.